bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -91,7 +91,11 @@ void ListInstances(bigtable::InstanceAdmin instance_admin, int argc,
 //! [get instance]
 void GetInstance(bigtable::InstanceAdmin instance_admin, int argc,
                  char* argv[]) {
-  // TODO(#419) implement tests and examples for GetInstance
+  if (argc != 2) {
+    throw Usage{"get-instance: <project-id> <instance-id>"};
+  }
+  std::string instance_id = ConsumeArg(argc, argv);
+  instance_admin.GetInstance(instance_id);
```

Comment:
Can you add something to print the instance?  Maybe convert it to a string using `google::protobuf::TextFormat` and print that string?  For an example see:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/master/bigtable/examples/bigtable_samples.cc#L60

Reply:
if I am correct, this is like below ? 
std::string instance_detail;
  google::protobuf::TextFormat::PrintToString(instance_admin.GetInstance(instance_id), &instance_detail);
  std::cout << "GetInstance details : " << instance_detail << std::endl;

Reply:
I would create a variable for the result of `instance_admin.GetInstance(instance_id)`.  Normally we try to avoid unnecessary variables, but this is sample code, the main goal is to make it easy to read.